### PR TITLE
Revamp landing page

### PR DESCRIPTION
**Pull request by chrisalice**
Totally redo [GitHub Pages site](http://freebuilder.inferred.org/) with up-front quickstart (including drop-down to select build environment), animated example code, javadoc links and tooltips. No longer attempting to reproduce the README with nicer fonts!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,12 +180,12 @@
   Person_Builder base = (Person_Builder) template;
   Person_Builder _defaults = <java-keyword>new</java-keyword> Person.Builder();
   <java-keyword>if</java-keyword> (!base.<java-field>_unsetProperties</java-field>.contains(Person_Builder.Property.NAME)
-      && (_defaults.<java-field>_unsetProperties</java-field>.contains(Person_Builder.Property.NAME)
+      &amp;&amp; (_defaults.<java-field>_unsetProperties</java-field>.contains(Person_Builder.Property.NAME)
```
Comment (hdurer) [inline index.html]: Why did you encode only these and not the ampersands in the hrefs above?

Comment (chrisalice) [inline index.html]: Whoops! Fixed.

**Comment by hdurer** [comment]
I find this way too busy.  To much stuff changing on the page making it hard to concentrate on the items.

On the plus side: seems to render well on narrow screens.

**Comment by hdurer** [comment]
Much nicer on the eye.

I like it.

**Pull request merged**